packages/feeds-client/src/test-utils/response-generators.ts

```diff
@@ -414,17 +414,22 @@ export function generateBookmarkUpdatedEvent(
 export function generateCommentAddedEvent(
   overrides: Omit<
     Partial<EventPayload<'feeds.comment.added'>>,
-    'comment' | 'type'
-  > & { comment?: Parameters<typeof generateCommentResponse>[0] } = {},
+    'comment' | 'activity' | 'type'
+  > & {
+    comment?: Parameters<typeof generateCommentResponse>[0];
+    activity?: Parameters<typeof generateActivityResponse>[0];
+  } = {},
```

Comment:
These changes are not related, just seen the errors and decided to fix them - I have to complete the `tsconfig.json` revamp PR so tests fail when typechecks fail.